integrations/snyk/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+## 0.1.93 (2024-10-08)
+
+
+### Improvements
+
+- Added ignore issue default action
```

Comment:
Let's be more descriptive here. something like Added a default Port self service action to allow users to ignore snyk vulnerabilities

Reply:
Done